sdk/cognitivelanguage/azure-ai-language-conversations-authoring/tests/test_cancel_training_job.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=line-too-long,useless-suppression
+import functools
+
+from devtools_testutils import AzureRecordedTestCase, PowerShellPreparer, recorded_by_proxy
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.language.conversations.authoring import ConversationAuthoringClient
+from azure.ai.language.conversations.authoring.models import TrainingJobResult
+
+ConversationsPreparer = functools.partial(
+    PowerShellPreparer,
```

Comment:
The updated name for this class is `EnvironmentVariableLoader`; `PowerShellPreparer` is [supported for backwards compatibility](https://github.com/Azure/azure-sdk-for-python/blob/9958caf6269247f940c697a3f982bbbf0a47a19b/eng/tools/azure-sdk-tools/devtools_testutils/__init__.py#L58), but the old name is deprecated.
```suggestion
    EnvironmentVariableLoader,
```

Reply:
Have updated. Thank you!